checkov/common/checks_infra/checks_parser.py

```diff
@@ -201,6 +202,7 @@ def parse_raw_check(self, raw_check: Dict[str, Dict[str, Any]], **kwargs: Any) -
         check.name = raw_check.get("metadata", {}).get("name", "")
         check.category = raw_check.get("metadata", {}).get("category", "")
         check.frameworks = raw_check.get("metadata", {}).get("frameworks", [])
+        check.severity = get_severity(raw_check.get("metadata", {}).get("severity", ""))
```

Comment:
Could you please check first if the severity exists?
It is better to have this field just if we have a value for it, and not always empty string.

Reply:
Ok. I just added a condition